specification/iotcentral/data-plane/Microsoft.IoTCentral/preview/2022-10-31-preview/iotcentral.json

```diff
@@ -0,0 +1,9326 @@
+      "x-ms-discriminator-value": "cloudProperty",
+      "allOf": [
+        {
+          "$ref": "#/definitions/JobData"
+        },
+        {
+          "$ref": "#/definitions/CapabilityJobData"
+        }
+      ]
+    },
+    "Collection": {
+      "description": "The collection of entities.",
+      "type": "object",
+      "properties": {
+        "value": {
+          "description": "The collection of entities.",
+          "type": "array",
+          "items": {
+            "type": "object"
+          }
+        },
+        "nextLink": {
+          "description": "URL to get the next page of entities.",
+          "type": "string"
+        }
+      },
+      "required": [
+        "value"
+      ]
+    },
+    "CommandConfiguration": {
+      "description": "Configuration specifying options for a command tile.",
+      "x-ms-discriminator-value": "command",
+      "allOf": [
+        {
+          "$ref": "#/definitions/TileConfiguration"
+        }
+      ],
+      "type": "object",
+      "properties": {
+        "device": {
+          "type": "string",
+          "description": "The device id that the command is associated with"
+        },
+        "command": {
+          "type": "string",
+          "description": "The command id to associate the tile to"
+        }
+      },
+      "required": [
+        "device",
+        "command"
+      ]
+    },
+    "CommandJobData": {
+      "description": "The command job data definition.",
+      "x-ms-discriminator-value": "command",
+      "allOf": [
+        {
+          "$ref": "#/definitions/JobData"
+        },
+        {
+          "$ref": "#/definitions/CapabilityJobData"
+        }
+      ]
+    },
+    "CommandTileConfiguration": {
+      "description": "Configuration specifying options for a command tile",
+      "x-ms-discriminator-value": "command",
+      "allOf": [
+        {
+          "$ref": "#/definitions/TileConfiguration"
+        }
+      ],
+      "type": "object",
+      "properties": {
+        "group": {
+          "description": "The ID of the device group to display",
+          "type": "string"
+        },
+        "command": {
+          "description": "The command to reference in the tile",
+          "type": "string"
+        },
+        "device": {
+          "description": "The device to reference in the tile"
+        }
+      },
+      "required": [
+        "group",
+        "command",
+        "device"
+      ]
+    },
+    "CountQueryRangeConfiguration": {
+      "description": "Configuration specifying the number of data points to query for a tile.",
+      "x-ms-discriminator-value": "count",
+      "type": "object",
+      "allOf": [
+        {
+          "$ref": "#/definitions/QueryRangeConfiguration"
+        }
+      ],
+      "properties": {
+        "count": {
+          "description": "The maximum number of data points to query for.",
+          "type": "integer",
+          "format": "int32",
+          "minimum": 100,
```

Comment:
So 100 is the only value?

Reply:
![image](https://user-images.githubusercontent.com/14871527/197583008-54ae9de2-f120-46dd-8cff-22cd69425dd8.png)

We had some limitations that have to align API with our UX, so 100 is fixed right now, putting other numbers will cause UX display issues.